packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -29,112 +29,108 @@ import {
 import i18next, {i18n} from 'i18next';
 import Backend, {BackendOptions} from 'i18next-http-backend';
 
+export type InitializeOptions = Pick<
```

Comment:
I would suggest `InitializationOptions` for the type name, starting with a noun instead of a verb, similar to `ConfigurationOptions` below (not `ConfigureOptions`).